pkg/cmd/server/admin/create_nodeconfig.go

```diff
@@ -110,13 +111,15 @@ func NewCommandNodeConfig(commandName string, fullName string, out io.Writer) *c
 }
 
 func NewDefaultCreateNodeConfigOptions() *CreateNodeConfigOptions {
+	defaulter := defaults.NewDefaults()
+
 	options := &CreateNodeConfigOptions{SignerCertOptions: &SignerCertOptions{}}
-	options.VolumeDir = "openshift.local.volumes"
```

Comment:
This should be coming from the config defaulting mechanism, so it's not technically wrong.  However, the defaulter should be returning a server API object, not some new mechanism.